e2e-test/java/io/cdap/plugin/slt/stepsdesign/StepDefinitionSLT.java

```diff
@@ -0,0 +1,472 @@
+    public void setDummyMTID() {
+        mtId = "02K";
+    }
+
+
+    @Given("^Open CDF replication and initiate pipeline creation")
+    public void open_cdf_replication() throws IOException, InterruptedException {
+        openCdf();
+        CDFDriver.get("http://localhost:11011/cdap/ns/default/replication/create");
+        sltActions.name_fill(UUID.randomUUID().toString().replaceAll("-", ""));
+        sltActions.next_click();
+        isErrorPresent = false;
+        bqRecordCount = 0;
+        sapRecordsCount = 0;
+    }
+
+    @When("^Source is SAP SLT fill connection parameters$")
+    public void source_is_sap_slt() throws IOException, InterruptedException {
+        sltActions.sltplugin_click();
+        sltActions.required_config(
+                "gcpProjectId",
+                "gcsDataPath",
+                mtId,
+                "sapJcoLibGcsPath",
+                "jcoclienthost",
+                "jcoclient",
+                "jcoclientsysnr",
+                "jcoclientuser",
+                "jcoclientpasswd");
+    }
+
+    @When("^Source is SAP SLT$")
+    public void source_is_sapslt() throws IOException, InterruptedException {
+        sltActions.sltplugin_click();
+    }
+
+    @Then("^User is able to set SLT parameter (.+) as (.+) and getting row (.+) for wrong input$")
+    public void userIsAbleToSetSLTParameterAsAndGettingRowForWrongInput(
+            String inputParameter, String inputValue, String errorMessage) {
+        errorExist = false;
+        elementHelper.replaceElementValue(sltLocators.inputParameter(inputParameter), inputValue);
+        sltActions.next_click();
+        errorExist = CDAPUtils.getErrorProp(errorMessage).
+                toLowerCase().contains(SLTLocators.rowError.getText().toLowerCase());
+        color = sltActions.rowError_color();
+        Assert.assertTrue(errorExist);
+        BeforeActions.scenario.write("Color of the text box" + color);
+        Assert.assertTrue(color.toLowerCase().contains("rgb(209, 86, 104)"));
+    }
+
+    @Then("^User is able to set SLT parameter \"([^\"]*)\" as \"([^\"]*)\"$")
+    public void userIsAbleToSetSLTParameter(
+            String inputParameter, String inputValue) {
+        elementHelper.replaceElementValue(sltLocators.inputParameter(inputParameter), inputValue);
+    }
+
+    @Then("^Replicate Existing Data is set to false$")
+    public void replicate_existing_data() {
+        sltActions.replicateExistingDataClick();
+        assertionHelper.verifyElementContainsText(sltLocators.replicateExistingData,
+                "No");
+    }
+
+    @Then("^Suspend Slt Job is set to true$")
+    public void suspend_existing_job() {
+        sltActions.suspendSltJobClick();
+        assertionHelper.verifyElementContainsText(sltLocators.suspendSltJob,
+                "Yes");
+    }
+
+    @Then("^Click on next$")
+    public void click_next() {
+        sltActions.next_click();
+    }
+
+    @Then("^Select Table \"([^\"]*)\"$")
```

Comment:
Why are u using regular expressions. 
You can simplify it like - > @Then("Select Table {string}")

Applies to all